src/MainWindow.cpp

```diff
@@ -1112,7 +1114,19 @@ void MainWindow::checkAutoStart()
 {
     QSettings s;
 
-    bool en = s.value("settings/auto_start").toBool();
```

Comment:
s.value() supports default value.
Easier to just do:

bool en = s.value("settings/auto_start", true).toBool();